index.bs

```diff
@@ -1830,6 +1831,106 @@ To <dfn>get the browsing context info</dfn> given |context|,
 
   1. Return |contexts info|
 
+</div>
+
+<div algorithm>
+To <dfn>await a navigation</dfn> given |context|, |request| and optionally
+|history handling| (default: "<code>default</code>") and |ignore cache| (default: false):
+
+1. Let |navigation id| be the string representation of a
+   [[!RFC4122|UUID]] based on truly random, or pseudo-random numbers.
+
+1. [=Navigate=] |context| with resource |request|, and using |context| as the
+   [=source browsing context=], with navigation id |navigation id|, and history
```

Comment:
The name of the parameter in https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate is `navigationId`, I think we should either use that or link the concept:

```suggestion
   [=source browsing context=], with <var ignore>navigationId</var> |navigation id|, and history
```

or

```suggestion
   [=source browsing context=], with [=navigation id=] |navigation id|, and history
```

Same for `historyHandling`.